tests/simpleTest/simple_test_client.hpp

```diff
@@ -171,14 +171,17 @@ class SimpleTestClient {
         char replyBuffer[kReplyBufferLength];
```

Comment:
Create a .cpp file and move the implementation in that file.

Reply:
Fixed